regress/pesterTests/Authorized_keys_fileperm.Tests.ps1

```diff
@@ -102,73 +84,73 @@ Describe "Tests for authorized_keys file permission" -Tags "CI" {
             #setup to have admin group as owner and grant it full control            
             Repair-FilePermission -Filepath $authorizedkeyPath -Owner $adminsSid -FullAccessNeeded $adminsSid,$systemSid -confirm:$false
 
             #Run
-            Start-SSHD-TestDaemon -Arguments "-d -p $port -o `"AuthorizedKeysFile .testssh/authorized_keys`" -E $logPath"
+            Start-SSHDTestDaemon -WorkDir $opensshbinpath -Arguments "-d -p $port -o `"AuthorizedKeysFile .testssh/authorized_keys`" -E $logPath"
             $o = ssh -p $port $ssouser@$server -o "UserKnownHostsFile $testknownhosts"  echo 1234
-            Stop-SSHD-TestDaemon
+            Stop-SSHDTestDaemon
             $o | Should Be "1234"
-            
         }
 
-        It "$tC.$tI-authorized_keys-positive(authorized_keys is owned by admins group and pwd have explict ACE)" {
+        It "$tC.$tI-authorized_keys-positive(authorized_keys is owned by admins group and pwd have explict ACE)"  -skip:$skip {
             #setup to have admin group as owner and grant it full control
             Repair-FilePermission -Filepath $authorizedkeyPath -Owner $adminsSid -FullAccessNeeded $adminsSid,$systemSid,$objUserSid -confirm:$false
 
             #Run
-            Start-SSHD-TestDaemon -Arguments "-d -p $port -o `"AuthorizedKeysFile .testssh/authorized_keys`" -E $logPath"
+            Start-SSHDTestDaemon -WorkDir $opensshbinpath -Arguments "-d -p $port -o `"AuthorizedKeysFile .testssh/authorized_keys`" -E $logPath"
             $o = ssh -p $port $ssouser@$server -o "UserKnownHostsFile $testknownhosts"  echo 1234
-            Stop-SSHD-TestDaemon
-            $o | Should Be "1234"
-            
+            Stop-SSHDTestDaemon
+            $o | Should Be "1234"          
         }
 
-        It "$tC.$tI-authorized_keys-negative(authorized_keys is owned by other admin user)" {
+        It "$tC.$tI-authorized_keys-negative(authorized_keys is owned by other admin user)"  -skip:$skip {
             #setup to have current user (admin user) as owner and grant it full control
             Repair-FilePermission -Filepath $authorizedkeyPath -Owner $currentUserSid -FullAccessNeeded $adminsSid,$systemSid -confirm:$false
 
             #Run
-            Start-SSHD-TestDaemon -Arguments "-d -p $port -o `"AuthorizedKeysFile .testssh/authorized_keys`" -E $logPath"
+            Start-SSHDTestDaemon -WorkDir $opensshbinpath -Arguments "-d -p $port -o `"AuthorizedKeysFile .testssh/authorized_keys`" -E $logPath"
             ssh -p $port -E $filePath -o "UserKnownHostsFile $testknownhosts" $ssouser@$server echo 1234
             $LASTEXITCODE | Should Not Be 0
-            Stop-SSHD-TestDaemon
-            $logPath | Should Contain "Authentication refused."
+            Stop-SSHDTestDaemon                  
+            $filePath | Should Contain "Permission denied"
+            $logPath | Should Contain "Authentication refused."            
```

Comment:
why $logpath as line 146 is logging into $filepath.

Reply:
$logpath is sshd log, $filePath is the client side log

Reply:
renamed to sshlog and sshdlog